platform/viewer/public/config/default.js

```diff
@@ -115,6 +115,11 @@ window.config = {
       label: 'W/L Preset 9',
       keys: ['9'],
     },
+    {
+      commandName: 'downloadAndZipSeriesOnViewports',
```

Comment:
@swederik, I left it here (along with the import extension on the "platform/viewer/src/index.js" file) in order to make the feature testable by the client. My plan is to remove it after the feature is approved and before it hits the master branch. Please let me know if you don't think it's necessary.